index.js

```diff
@@ -1,6 +1,10 @@
 const qs = require('qs');
 const result = qs.parse(document.location.search.substring(1));
-window.render = (html) => document.getElementById("playground").innerHTML = html;
+
+System.config({
+    transpiler: "babel"
```

Comment:
FYI: transpiler can be located in package.json, probably I'll move it in the nearest future.